modules/organization-policy/variables.tf

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "config_directory" {
+  description = "Paths to a folder where organization policy configs are stored in yaml format. Files suffix must be `.yaml`."
+  type        = string
+  default     = null
+}
+
+# TODO: convert to a proper data structure map(map(object({...}))) once tf1.3 is released and optional object keys are avaliable,
```

Comment:
@ludoo @juliocc WDYT, I fould this as an easiest way to allow uses skip attributes while defining policies (like avoid tons of keys set to null), and the same time there won't be a breaking change when tf1.3 released and we define a proper data structure for the variable. 
Do we want going the same way for org policies in project, folder, org modules?

Reply:
> @ludoo @juliocc WDYT, I found this as the easiest way to allow uses skip attributes while defining policies (like avoid tons of keys set to null), and the same time there won't be a breaking change when tf1.3 released and we define a proper data structure for the variable.

I agree on the null keys point but you know we avoid variables with `type=any`. We actually started thinking how to migrate to the these new org policy resource and put it aside until optionals are GA because we didn't find a reasonable way to keep type information and not forcing the user to pass an object with a ton of null values.

> Do we want going the same way for org policies in project, folder, org modules?

I'd say yes but I can be convinced with good arguments. The main drawback I see with the new approach is that you have to pass the resource id as keys (e.g. the folder id), which will certainly break once you start trying to apply policies to folders/projects created by another module.

That being said, I really like your idea to pass policies as yaml files, +1000 to that. Maybe we need both things (standalone and org/folder/project support)?

@ludoo wdyt?

Reply:
@juliocc 
>I agree on the null keys point but you know we avoid variables with type=any. We actually started thinking how to migrate to the these new org policy resource and put it aside until optionals are GA because we didn't find a reasonable way to keep type information and not forcing the user to pass an object with a ton of null values.
That's why I wanted to know what you think, my worry is that tf1.3 is still in alpha stage, but we need to provide users with a way to use conditions and multiple rules for org-policies, so I thought having type=any + a good documentation could be acceptable while we are waiting for optional keys.
> I'd say yes but I can be convinced with good arguments. The main drawback I see with the new approach is that you have to pass the resource id as keys (e.g. the folder id), which will certainly break once you start trying to apply policies to folders/projects created by another module.
There won't be keys with resource IDs for project/org/folder since there will be only one resource which is created by the module. When I'm asking around the same approach I mean the var format and type=any thing, but keys are going to be constants IDs which is static.

Reply:
I absolutely agree we should not have `any` as a variable type. I would

- try to simplify the interface further
- come up with proper types
- possibly start implementing optionals in this module, it's new we can require preview features on this, we're not breaking compatibility

+1 on yaml files, Aleks you're the father of factories, it makes sense it's you to introduce it for org policies! :)

Reply:
Experimental feature used to avoid `type=any`